model/enduser/registry.yaml

```diff
@@ -0,0 +1,32 @@
+groups:
+  - id: registry.enduser
+    type: attribute_group
+    display_name: End User Attributes
+    brief: >
+      Describes information about the end user, which can be used as a subdomain of browser, client, or user domains.
+    attributes:
+      - id: enduser.id
+        type: string
+        stability: experimental
+        brief: >
+          Identifier of an end user who interacts with a system.
+          This identifier may be unique only through best-effort means and does not imply that the user is authenticated to the system.
+        examples: ['QdH5CAWJgqVT4rOr0qtumf']
+      - id: enduser.role
+        type: string
+        stability: experimental
+        brief: 'Actual/assumed role the client is making the request under extracted from token or application security context.'
+        examples: 'admin'
+      - id: enduser.scope
```

Comment:
same here, do we need it right now? Can we keep it deprecated for the time being until it's necessary?

Reply:
will change it to `deprecated` for now.